person.rb

```diff
@@ -1,26 +1,85 @@
-require_relative 'nameable'
+class Nameable
+  attr_accessor :name
+
+  def initialize(name)
+    @name = name
+  end
+
+  def correct_name
+    raise NotImplementedError, 'Kindly Implement correct_name method to return the correct name'
+  end
+end
```

Comment:
- [ ] While you make changes you put all classes in one file and this is not a best practice, so separate them again and also for `BaseDecorator`, `TrimmerDecorator` , and `CapitalizeDecorator`

Reply:
@omarsalem7 You Guys I don't know the reason for different review, one said other another said another thing. 😠

Reply:
@omarsalem7 I tried to fix it based on yours also.